test/integration/controller_binding_test.go

```diff
@@ -416,7 +416,9 @@ func TestCreateServiceBindingWithParameters(t *testing.T) {
 		},
 	}
 	for _, tc := range cases {
+		tc := tc
```

Comment:
is this line really necessary? we shouldn't be altering tc in this loop, right?

Reply:
Yes. It is necessary to capture the current scope.

For reference https://blog.golang.org/subtests

Reply:
Ahh, okay.  Thanks for the link.